### ISSUE TITLE
Attributes skipped while importing json

### ISSUE THREAD

**Issue by urswiss** [mongodb-js/compass#3373]
#### Current Behavior

The attribute `frameworkContract` is ignored while importing. There is no error message.
To reproduce: Click on `Add data`, `Import File`, `Select a file...`, open the file which contains the data below. The option `Stop on errors` is not checked.

#### Code/Gist

```
[{
  "_id": "12345678",
  "linkName": "any link name",
  "defaultConfiguration": {
    "frameworkContract": {
      "frameworkContract": "123",
      "memberNumber": "any member number"
    },
    "campaign": null
  }
}
]
```

#### Expected Behavior/Code

Anything in the json should end up in the collection.

#### Environment

- node: 16.16.0 / npm: 8.11.0
- OS: Windows Version 10.0.19044.1889
- MongoDB Compass Version: 1.32.6

**Comment by urswiss** [comment]
works fine in MongoDB Compass version 1.26.1

**Comment by Anemy** [comment]
Hi @urswiss this looks like this bug we're currently tracking in jira: https://jira.mongodb.org/browse/COMPASS-5971
We'll update here when there's a fix out. Thanks for reporting the issue.

**Comment by RogerMeyer** [comment]
It works for / with MongoDB Compass 1.30.1. I had no issues.

**Comment by Anemy** [comment]
@urswiss This is fixed in the newly released version `1.33.1`. Thanks again for reporting this and helping us improve Compass. Going to close this, feel free to reopen if you update and things aren't behaving as expected.

https://github.com/mongodb-js/compass/releases
